sceneManager.js

```diff
@@ -162,6 +177,22 @@ class SceneManager {
 
     update() {
         this.menuButtonTimer += this.game.clockTick;
+        if (this.time === 99) {
```

Comment:
Instead of manipulating your own custom timer, I would just use `this.game.clockTick` directly. `this.time` -> `this.timeLeft` for clarity?

```js
update() {
  // ...
  if (!this.title && !this.transition && !this.paused) {
    this.timeLeft -= this.game.clockTick;
    if (this.timeLeft.round() == 99) {
      // Play Music Logic
    }
  }
}
```

Now the code above is pseudo-ish. There is a change you double play (the starting plays two times), but I find it unlikely and not too disturbing as the double play with be within frames. If you want to be a bit more rigorous, you can have a flag to make sure it doesn't double play. I think this will fix your problem. Making the logic a bit more simple and having less state to keep track of (no need for `this.timer` anymore)